frontend/src/metabase/palette/components/PaletteResultItem.unit.spec.tsx

```diff
@@ -62,3 +70,108 @@ describe("PaletteResultItem", () => {
     expect(screen.queryByRole("img")).not.toBeInTheDocument();
   });
 });
+
+/** For some tests we need to render the PaletteResultsList so that the Enter key works */
+const setupInList = ({ item }: { item: Partial<PaletteActionImpl> }) => {
```

Comment:
Would it make sense to do these tests in `PaletteResultsList.unit.spec.tsx`?

Reply:
I originally put them there, but these aren't really tests of that component